File: leetcode/75 leetcode/two_pointers/move_zeros/move_zeros.c
void moveZeroes(int* nums, int numsSize){
    int l[numsSize];
    int i = 0, j = 0;
    while(i<numsSize){
        if (nums[i] != 0) {
            l[j] = nums[i];
            j++;
        }
        i++;
        
    }
    while(j<numsSize){
        l[j] = 0;
        j++;
    }
    i = 0;
    while(i<numsSize){
        nums[i] = l[i];
        i++;
    }

}

File: leetcode/75 leetcode/two_pointers/move_zeros/move_zeros.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        l = [i for i in nums if i != 0]
        l.extend([i for i in nums if i == 0])
        for i,x in enumerate(l):
            nums[i] = x